config/params/loader.go

```diff
@@ -65,6 +65,8 @@ func UnmarshalConfig(yamlFile []byte, conf *BeaconChainConfig) (*BeaconChainConf
 	}
 	// recompute SqrRootSlotsPerEpoch constant to handle non-standard values of SlotsPerEpoch
 	conf.SqrRootSlotsPerEpoch = primitives.Slot(math.IntegerSquareRoot(uint64(conf.SlotsPerEpoch)))
+	// Recompute the fork schedule
+	conf.InitializeForkSchedule()
```

Comment:
isn't this already called when a new config set is initialized?

Reply:
Yes but the point is that this function explicitly may overwrite the fork epochs and versions and the maps are left unmodified in inconsistent state